python/setup.py

```diff
@@ -20,60 +20,13 @@
 
 SCRIPT_DIR = os.path.dirname(__file__)
 
-
```

Comment:
You don't need to remove them because we will need it in the future when configuring the version.
If any line/function that causes problems for the current flow, you could comment it out with FIXME.